engine.py

```diff
@@ -955,22 +955,14 @@ def __get_wrap_instance_method(self):
 
         This checks Qt versions in order of:
 
-            1. PySide / shiboken
-            2. PySide2 / shiboken2
-            3. PySide6 / shiboken6
+            1. PySide2 / shiboken2
+            2. PySide6 / shiboken6
 
         This method is currently only necessary to get the VRED main window. If more cases
         arise such that the correct shiboken module is found for the current Qt version, this
         should be provided by the tk-core QtImporter.
         """
 
-        try:
-            from shiboken import wrapInstance
```

Comment:
Yes, not sure what the exact tk-core version that introduced it, but now you can access the shiboken module like:
```
from sgkt.platform.qt import shiboken

shiboken.wrapInstance
```

Reply:
> Yes, not sure what the exact tk-core version that introduced it

It's tk-core v0.21.1


> Note that PyQt (not PySide) does not include shiboken

As we are dropping the compatibility with Qt4 (PySide/PyQt4), we no longer support any PyQt interface. We do not support PyQt5 as far as I know. Only PySide2.

Reply:
Why is this not failing on PySide2? If I recall correctly it's called `shiboken2` there.

Reply:
Are you referring to this?

```
from sgkt.platform.qt import shiboken

shiboken.wrapInstance
```

This works because `qt.shiboken` property will provide the correct shiboken module based on the version of PySide that was imported (e.g. it will give you `shiboken2` if using PySide2 and `shiboken6` for PySide6).